src/WebJobs.Script/Binding/ExtensionBinding.cs

```diff
@@ -70,6 +71,12 @@ public override Collection<CustomAttributeBuilder> GetCustomAttributes(Type para
                     context.Value = result;
                 }
             }
+            else if (_binding.DefaultType == typeof(JArray))
+            {
+                JArray entityArray = await context.Binder.BindAsync<JArray>(_attributes.ToArray());
+                string json = entityArray.ToString(Formatting.None);
```

Comment:
good catch. I was copying over code from the previous binding; but it looks like we don't need this. 

---
In reply to: [85890556](https://github.com/Azure/azure-webjobs-sdk-script/pull/858#discussion_r85890556) [](ancestors = 85890556)